charts/atlantis/values.yaml

```
@@ -731,6 +731,9 @@ redis: {}
 # -- When managing secrets outside the chart for the Redis secret, use this variable to reference the secret name.
 redisSecretName: ""
 
+# -- Key within the existing Redis secret that contains the password value.
+redisSecretPasswordKey: "password"
```

Nit: to keep it similar to other keys, we only quote empty values:
```suggestion
redisSecretPasswordKey: password
```